src/jdk.incubator.jextract/share/classes/jdk/internal/jextract/impl/FunctionalInterfaceBuilder.java

```diff
@@ -69,7 +69,7 @@ private String parameterName(int i) {
         if (parameterNames.isPresent()) {
             name = parameterNames.get().get(i);
         }
-        return name.isEmpty()? "_x" + i : name;
+        return name.isEmpty()? "_x" + i : Utils.javaSafeIdentifier(name);
```

Comment:
Do we do this for regular native functions?

e.g.
```
int foo(int abstract);
```

Reply:
We already call Utils.javaSafeIdentifier() elsewhere. So this is already taken care of (8262825)

Reply:
thanks for the explanation